Microsoft.Azure.CosmosRepository.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
```

Comment:
Sorry about this file is it because I'm using Visual Studio 2022?

I'll make sure to clean it up if it's a problem

Reply:
No worries, I already upgraded this to 2022